README.md

```diff
@@ -23,11 +23,24 @@
 # Fingerprint Pro Cloudflare worker
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
 
+## Requirements
+
+This library needs these to be in order to build and run:
+
+- Node 16 or Node 18
+- Typescript 4
+- Playwright (with a headless browser preferably chromium)
+- [Wrangler](https://developers.cloudflare.com/workers/wrangler/install-and-update/)
```

Comment:
There are different `wrangler` versions out there, `v1`, `v2` and `v3`. In my case, I started using `v1` and then switched to `v2`, and never tried `v3`. We probably support `v2` and `v3` and not `v1` but need to make sure and mention it here.

Reply:
We support v3+ I mentioned in the readme